docs-crawler/docs/aggregate_crawler_output.py

```diff
@@ -40,13 +46,17 @@ def load_base_defs(self):
         self.populate_command('torch')
 
     def hydrate_args(self, base_args, base_kwargs):
+        base_args = list(filter(lambda a: a not in ['', '?'], base_args))
```

Comment:
We had lots of arguments with empty named and only question marks due to strange formatting of a few commands on the page. I am just stripping these out for now. In the future, we will want to just remove all support for these commands.